CMakeLists.txt

```diff
@@ -1,14 +1,18 @@
 cmake_minimum_required(VERSION 3.1...3.14)
 
-project(face_detector VERSION 3.11.3 LANGUAGES CXX)
+project(face_detector VERSION 3.11.5 LANGUAGES CXX)
 
 set(MAIN_PROJECT OFF)
 
-# Optionally build the example
+# Optionally build the examples
 option(CLFML_FACE_DETECTOR_BUILD_EXAMPLE_PROJECTS "Build example projects" ON)
+
 # Optionally enable edge tpu support (Requires libedgetpu library to be installed!)
 option(CLFML_FACE_DETECTOR_ENABLE_CORAL_TPU "Enable CORAL EDGE_TPU support" OFF)
 
+# Optionally enable ROS2 package build (Requires ROS2 jazzy to be installed!)
+option(CLFML_ROS2_PACKAGE_BUILD "Build a ROS2 package" ON)
```

Comment:
It might be better to leave this value to default OFF. As not everyone using this library will use ROS. By setting this as default value ON, we force the users of our library to have the ROS packages installed regardless of whether they are using ROS or not.

I would highly suggest making it a opt-in option, such as with the CORAL support